### ISSUE TITLE
Unable to navigate to legend with accessibility module on IE

### ISSUE THREAD

**Issue by oysteinmoseng** [highcharts/highcharts#6460]
#### Expected behaviour
Should be able to TAB to legend items.

#### Actual behaviour
Browser tabs through the series paths instead, apparently.

#### Live demo with steps to reproduce
http://jsfiddle.net/gh/get/jquery/1.7.2/highcharts/highcharts/tree/master/samples/highcharts/accessibility/accessible-line/

#### Affected browser(s)
IE (verified with IE11)

**Comment by oysteinmoseng** [comment]
Follow this up in #6511 instead.
